sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/SnowflakeIO.java

```diff
@@ -1192,7 +1192,13 @@ private String quoteField(String field) {
     }
 
     private String quoteField(String field, String quotation) {
```

Comment:
consider rename this private method (and its overload above) to sth like "quoteNonEmptyField" for clarity

Reply:
fixup 016e641396f4a5aad0d672a5d8e1de92a43eafa1